components/authentication/base/admin-checker/cronjob.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: batch/v1
```

Comment:
Can you also remove this from the development overlay? 
Base will give us staging + production.  We don't want notifications from dev clusters.

Reply:
This is already the case, the whole authentication component is removed in the development overlay: https://github.com/redhat-appstudio/infra-deployments/blob/main/argo-cd-apps/overlays/development/delete-applications.yaml#L5